flink-libraries/flink-table/src/test/scala/org/apache/flink/table/api/scala/stream/sql/SqlITCase.scala

```diff
@@ -317,4 +397,24 @@ class SqlITCase extends StreamingWithStateTestBase {
     result.addSink(new StreamITCase.StringSink)
     env.execute()
   }
+
+}
+
+object SqlITCase {
+
+  class TimestampWithLatenessWatermark(allowedLateness: Long)
```

Comment:
@sunjincheng121 can you please consider changing the allowedLateness to offset:
`class AssignerWithWatermarkOffset(watermarkOffset: Long)`